File: app/src/main/java/me/blog/korn123/easyphotomap/models/ThumbnailItem.java
package me.blog.korn123.easyphotomap.models;

import android.net.Uri;

/**
 * Created by CHO HANJOONG on 2016-08-02.
 */
public class ThumbnailItem {
    public String imageId;
    public String imagePath;
    public String thumbnailPath;
    public Uri thumbnailUri;
    public Uri imageUri;

    public ThumbnailItem(){}

    public ThumbnailItem(String imageId, String imagePath, String thumbnailPath) {
        this.imageId = imageId;
        this.imagePath = imagePath;
        this.thumbnailPath = thumbnailPath;
    }
}

File: app/src/main/java/me/blog/korn123/easyphotomap/models/ThumbnailItem.kt
package me.blog.korn123.easyphotomap.models

import org.apache.commons.lang.StringUtils

/**
 * Created by CHO HANJOONG on 2016-08-02.
 */
class ThumbnailItem(imageId: String, imagePath: String, thumbnailPath: String) {

    var imageId: String = StringUtils.EMPTY
    var imagePath: String = StringUtils.EMPTY
    var thumbnailPath: String = StringUtils.EMPTY

    init {
        this.imageId = imageId
        this.imagePath = imagePath
        this.thumbnailPath = thumbnailPath
    }

}
